contracts/pods/PodAdminGateway.sol

```diff
@@ -157,6 +157,19 @@ contract PodAdminGateway is CoreRef, IPodAdminGateway {
         emit PodMembershipTransferLock(_podId, _lock);
     }
 
+    /// @notice Transfer the admin of a pod to a new address
+    /// @dev Permissioned to GOVERNOR, POD_ADMIN and the specific pod admin role
+    function transferAdmin(uint256 _podId, address _newAdmin)
+        external
+        hasAnyOfThreeRoles(TribeRoles.GOVERNOR, TribeRoles.POD_ADMIN, getSpecificPodAdminRole(_podId))
+    {
+        ControllerV1 podController = ControllerV1(memberToken.memberController(_podId));
+        address oldPodAdmin = podController.podAdmin(_podId);
```

Comment:
I recommend emitting the event first, then you don't have to store this in memory.

Reply:
It'll get stored on the stack so will be cheap